### ISSUE TITLE
Access Method List on Download Confirmation

### ISSUE THREAD

**Issue by MRichards99** [ral-facilities/datagateway#800]
**Description**:
On Download, the drop down list element of the download confirmation form has a couple of issues. When focusing on the list with a screen reader, it duplicates the currently selected access method. "clickable  HTTPS HTTPS  menu button  subMenu" (extracted from NVDA's speech viewer). This seems to happen no matter the access method currently selected (i.e. either https or globus). This duplication should be removed.  It would be good if this screen reader output could also be changed to tell the user that it's for access method selection - the current screen reader output doesn't make this clear.

When using this drop down using a keyboard, it doesn't seem particularly easy to use. To access the options, you press the space bar which is standard practice. But to be able select a different option, you have to press the space bar before then using the arrow keys to change to the desired option. The space bar shouldn't need to be used, it could confuse the user.

This has been created as a result of #738 

**Acceptance criteria**:
- [x] Change the aria labels/screen reader output to remove duplication and make the function of the drop down clearer to the user
- [x] Keyboard navigation on the access method list shouldn't require use of the space bar so the option can be changed

**Comment by MRichards99** [comment]
There's a near identical issue on Search, but with a button (adding it here seems appropriate). `searchBox.search_button_arialabel` should have the word "button" removed from its entry in `default.json`
